data_pipeline/l2_job.py

```diff
@@ -9,8 +10,9 @@ def run_l2beat(config=None):
     with open(sys.argv[1]) as json_data:
         config = json.load(json_data)
 
+    parent_directory = os.path.dirname(os.getcwd())
     # create object to store data
-    output_object = create_storage(path=config['output'])
+    output_object = create_storage(path=parent_directory + config['output'])
```

Comment:
use `join` to concat paths: https://stackoverflow.com/questions/10918682/platform-independent-path-concatenation-using